sdk/python/feast/infra/offline_stores/bigquery.py

```diff
@@ -430,20 +430,16 @@ def _get_bigquery_client(project: Optional[str] = None):
  Thus we only need to compute the latest timestamp of each feature.
 */
 {{ featureview.name }}__latest AS (
-    SELECT
-        {{featureview.name}}__entity_row_unique_id,
-        MAX(event_timestamp) AS event_timestamp
-        {% if featureview.created_timestamp_column %}
-            ,ANY_VALUE(created_timestamp) AS created_timestamp
-        {% endif %}
-
+    SELECT *,
+        ROW_NUMBER() OVER(
+            PARTITION BY {{featureview.name}}__entity_row_unique_id
+            ORDER BY event_timestamp DESC{% if featureview.created_timestamp_column %},created_timestamp DESC{% endif %}
+        ) AS row_number,
```

Comment:
So is this table still the latest values? Wouldn't you have all the different rows with different row numbers?

Reply:
Done